docs/en/week08/08-3.md

```diff
@@ -0,0 +1,198 @@
+---
+lang-ref: ch.08-3
+title: Generative Models - Autoencoders
+lecturer: Alfredo Canziani
+authors: Nandhitha Raghuram, Xinyi Zhao
+date: 25 March 2021
+---
+
+### Recap: Predictive Latent Variable Model
+As we know latent variables are useful in providing additional information that help in predicting multiple values of the target $y$, assiociated with the input $x$. We have seen earlier, in order to predict the target variable $y$ from latent variable $z$, the energy function is given as,
+$$E(y,z) = [y_1 - g_1(z)]^2 + [y_2 - g_2(z)]^2, y \in Y$$
+where $g = [g_{1}, g_{2}]^{T}$, $\mathbb{R} \rightarrow \mathbb{R}^{2}$ and $g(z) = [w_{1}cos(z), w_{2}sin(z)]$.
+
+In this case, the the latent variable $z$ is one dimensional. However, if the $z$ has a higher dimension, this could lead to an energy function of $0$  everywhere. Hence, in order to ensure lower energy values for more compatible predictions we introduce a regularized factor. 
+
+**Training recap**
+
+Given an observation $y$, training the regualarized latent variable model
+$$
+E(y,z) = C(y,\tilde{y}) + R(z)
+$$ 
```

Comment:
You have not read [these instructions](https://atcold.github.io/pytorch-Deep-Learning/en/about/).
You need an empty line before and after an display equation. Please, render the website locally to verify corrected rendering. Instructions [here](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/README.md).

Reply:
done